web/server/codechecker_server/database/run_db_model.py

```diff
@@ -364,8 +372,20 @@ class Report(Base):
                                    'off',
                                    'unavailable',
                                    name='detection_status'))
+    review_status = Column(ReviewStatusType,
+                           nullable=False,
+                           server_default='unreviewed')
```

Comment:
We are using this column in every query related to review statuses (filtering, etc.) Having `NULL` here would complicate those queries, because `NULL` and `'unreviewed'` would both mean "unreviewed".